holoviews/core/spaces.py

```diff
@@ -589,6 +596,53 @@ def dynamicmap_memoization(callable_obj, streams):
         callable_obj.memoize = memoization_state
 
 
+
+class periodic(object):
+    """
+    Implements the utility of the same name on DynamicMap.
+
+    Used to defined periodic event updates that can be started and
+    stopped.
+    """
+    _periodic_util = util.periodic
+
+    def __init__(self, dmap):
+        self.dmap = dmap
+        self.instance = None
+
+    def __call__(self, period, count, param_fn=None):
+        """
+        Run a non-blocking loop that updates the stream parameters using
+        the event method. Runs count times with the specified period. If
+        count is None, runs indefinitely.
+
+        If param_fn is not specified, the event method is called without
+        arguments. If it is specified, it must be a callable accepting a
+        single argument (the iteration count) that returns a dictionary
+        of the new stream values to be passed to the event method.
+        """
+
+        if self.instance is not None and not self.instance.completed:
+            raise RuntimeError('Periodic process already running. '
+                               'Wait until it completes or call '
+                               'stop() before running a new periodic process')
+        def inner(i):
+            kwargs = {} if param_fn is None else param_fn(i)
+            self.dmap.event(**kwargs)
+
+        instance = self._periodic_util(period, count, inner)
+        instance.start()
+        self.instance= instance
+
+    def stop(self):
+        "Stop the periodic process."
+        self.instance.stop()
+
+    def __str__(self):
+        return "<holoviews.core.spaces.periodic method>"
+
```

Comment:
It isn't strictly necessary, but I think it is a bit nicer than the default which is something like:

```
<holoviews.core.spaces.periodic at 0x10886e850>
```

Instead of a random memory address, I say that it is a method which reflects how this utility is used (i.e it lives on ``DynamicMap``).

Happy to remove it if you think the default is better.